src/Widgets/TextWidget/TextWidgetEditingConfig.js

```diff
@@ -23,4 +23,18 @@ Scrivito.provideEditingConfig("TextWidget", {
   initialContent: {
     alignment: "left",
   },
+  validations: [
+    [
+      "text",
+
+      text => {
+        if (text.length === 0) {
+          return {
+            message: "The text should be set.",
+            severity: "warning",
+          };
```

Comment:
Drop it - to much for every text widget (it has no initial content)

Reply:
👍